src/lib.rs

```diff
@@ -0,0 +1,230 @@
+#![cfg_attr(test, feature(test))]
```

Comment:
Could you explain the rationale behind this?

Reply:
It simply allows me to benchmark my code by running `cargo +nightly bench` and when I need to compile my code normally, I can still run `cargo build --release` without needing to use the nightly compiler